### ISSUE TITLE
Import a6bfa4982a: Create tool emissive gizmos are locally available now (by Maki)

### ISSUE THREAD

**Issue by daleglass** [overte-org/tivolicloud#326]
This is an automated proposal to look at a commit made by Maki and import it into [Overte](https://github.com/overte-org/overte)

| **Commit:** |  a6bfa4982a93de68033d8aa4c29353d26c70b597 |
| --- | --- |
| **Author:** | Maki |
| **Date:** | Tue, 07 Apr 2020 17:15 |

```
Create tool emissive gizmos are locally available now

```

**Stats:**

| **Filename** | Stats | Lines | Added | Removed | Lines in blame |
| --- |  --- | --- | --- | --- | --- |
| scripts/system/create/assets/emissive.fs |  | 7 | 7 | 0 | ⚠ File gone
| scripts/system/create/entitySelectionTool/entitySelectionTool.js |  | 123 | 29 | 94 | ⚠ 0
| 2 files | - | 130 | 36 | 94 | 0 |

To work on this, please assign the issue to yourself, then look at the commit and decide whether this would be a good addition to Overte.

* If the commit is useful, tag it with "Tivoli: Keep", and keep it open until it's merged.
* If the commit is not useful, tag it with "Tivoli: Discard", and close it.
* If the commit is not useful right now, but might be later, tag it with "Tivoli: Maybe later", and close it.
* If it's hard to decide, tag it with "Tivoli: Discuss", and keep it open.

Useful commits should be submitted as a [PR against Overte](https://github.com/overte-org/overte/pulls). Tag this issue in PR, so that it's automatically closed once the PR is merged.

You can cherry-pick this issue with this command:

    git cherry-pick a6bfa4982a93de68033d8aa4c29353d26c70b597

**Comment by daleglass** [comment]
not entirely sure of what this is doing

**Comment by AleziaKurdis** [comment]
The gizmo is a type of entity that is used for the selection tool, to rotate the selection. 
The gizmo that tivoli talk about are "overlays", 
The create app (Vircadia and Overte) has been converted to use "Local Entities" instead of the deprecated "overlays".
The gizmo local entity support material entities so we have this solved to have them emissive.

We don't need this for the Create App.

@HifiExperiments can confirme, but I don't think we need this at all.
I think gizmo overlay is officially deprecated. and emissivity can be done using material for entities

**Comment by AleziaKurdis** [comment]
https://apidocs.overte.org/Overlays.html#.OverlayProperties-Circle3D

**Comment by HifiExperiments** [comment]
yep, no need for this, they’re using custom shaders on overlays which will be significantly less performant than materials on local entities
